service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/DiagnosticReportMapper.java

```diff
@@ -87,7 +89,8 @@ public String mapDiagnosticReportToCompoundStatement(DiagnosticReport diagnostic
             .extensionId(fetchExtensionId(diagnosticReport.getIdentifier()))
             .availabilityTimeElement(StatementTimeMappingUtils.prepareAvailabilityTime(diagnosticReport.getIssuedElement()))
             .narrativeStatements(reportLevelNarrativeStatements)
-            .specimens(mappedSpecimens);
+            .specimens(mappedSpecimens)
```

Comment:
I think the whole section (which confidentialityCode is part of) can be extracted into a separate method with the same name diagnosticReportCompoundStatementTemplateParameters.

Reply:
TODO separate clean-up PR, out of scope for NIAD-3144